### PR TITLE
[DEVX-131] Text_upload

### PR DESCRIPTION
**Pull request by sanjaychelliah**
### What

- Text upload from CSV, folder
- Creating app with workflow id

### Examples

```python
# Note: CLARIFAI_PAT must be set as env variable.
from clarifai.client.user import User
client = User(user_id="user_id")

# Create app and dataset
app = client.create_app(app_id="demo_app")
dataset = app.create_dataset(dataset_id="demo_dataset")

#upload text from csv
dataset.upload_from_csv(csv_path='csv_path', labels=True)

#upload data from folder
dataset.upload_from_folder(folder_path='folder_path', input_type='text', labels=True)

```

### Note

- While creating app, default workflow_id is set as 'Language-Understanding' for now(Temporary, will revert to empty workflow after hackathon)
- TODO: upload image_URLs from csv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,10 +30,17 @@ client = User(user_id="user_id")
 apps = client.list_apps()
 
 # Create app and dataset
-app = client.create_app(app_id="demo_app")
+app = client.create_app(app_id="demo_app", workflow_id="Universal")
```
Comment (sainivedh) [inline README.md:33]: Can we change this to `base_workflow`

File: clarifai/client/dataset.py [hunk 2]
```diff
@@ -220,6 +220,62 @@ def upload_dataset(self,
 
     self._data_upload(dataset_obj)
 
+  def upload_from_csv(self,
+                      csv_path: str,
+                      input_type: str = 'text',
+                      labels: bool = True,
+                      chunk_size: int = 128) -> None:
+    """Uploads dataset from a csv file.
+    Note: csv file should have either one(input) or two columns(input, labels).
```
Comment (sainivedh) [inline clarifai/client/dataset.py:229]: Can we have `Note` at the bottom

File: clarifai/client/dataset.py [hunk 3]
```diff
@@ -220,6 +220,62 @@ def upload_dataset(self,
...
+          csv_path=csv_path, dataset_id=self.id, labels=labels)
+    self.input_object._bulk_upload(inputs=input_protos, chunk_size=chunk_size)
+
+  def upload_from_folder(self,
+                         folder_path: str,
+                         input_type: str,
+                         labels: bool = False,
+                         chunk_size: int = 128) -> None:
+    """Upload dataset from folder.
+    Note: The filename is used as the input_id.
```
Comment (sainivedh) [inline clarifai/client/dataset.py:256]: `Note`

File: clarifai/client/user.py [hunk 4]
```diff
@@ -53,10 +57,13 @@ def create_app(self, app_id: str, **kwargs) -> App:
 
     Example:
         >>> from clarifai.client.user import User
-        >>> app = User("user_id").create_app("app_id")
+        >>> client = User(user_id="user_id")
+        >>> app = client.create_app(app_id="app_id",workflow_id="Universal")
```
Comment (sainivedh) [inline clarifai/client/user.py:61]: `base_workflow`

**Review by sainivedh — COMMENTED** [review]
Looks good. Some minor Comments

File: README.md (hunk 1 above)
Comment (sanjaychelliah) [reply README.md:33]: Yeah, more relevant 👍

**Review by sainivedh — APPROVED** [review]
LGTM

**Pull request merged**